packages/@aws-cdk/aws-apprunner-alpha/lib/observability-configuration.ts

```diff
@@ -0,0 +1,157 @@
+import * as cdk from 'aws-cdk-lib/core';
+import { Construct } from 'constructs';
+import { CfnObservabilityConfiguration } from 'aws-cdk-lib/aws-apprunner';
+
+/**
+ * The implementation provider chosen for tracing App Runner services
+ */
+export enum Vendor {
+  /**
+   * AWS X-RAY
+   */
+  AWSXRAY = 'AWSXRAY',
+}
+
+/**
+ * Properties of the AppRunner Observability configuration
+ */
+export interface ObservabilityConfigurationProps {
+  /**
+   * The name for the ObservabilityConfiguration.
+   *
+   * @default - a name generated by CloudFormation
+   */
+  readonly observabilityConfigurationName?: string;
+
+  /**
+   * The implementation provider chosen for tracing App Runner services.
+   *
+   * @default Vendor.AWSXRAY
```

Comment:
```suggestion
   * @default - tracing is not enabled for the service
```
From the CloudFormation [documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apprunner-observabilityconfiguration.html#cfn-apprunner-observabilityconfiguration-traceconfiguration), it seems we're allowed to omit this value.
Can you please verify if the construct deploys when that is the case?
Ie, changing the implementation to:
```ts
    const resource = new CfnObservabilityConfiguration(this, 'Resource', {
      observabilityConfigurationName: props.observabilityConfigurationName,
      traceConfiguration: props.vendor ? {
        vendor: props.vendor,
      } : undefined,
    });
```

Reply:
Thanks.
I fixed it, and I tried to deploy it.
An Observability Configuration without TraceConfiguration could be deployed.

```sh
% aws apprunner describe-observability-configuration --observability-configuration-arn arn:aws:apprunner:us-east-1: 01234567890:observabilityconfiguration/my-observability-no-vendor/1/2bc8fefaddcc4b878c0dfe2598412555 --region us-east-1
{
    "ObservabilityConfiguration": {
        "ObservabilityConfigurationArn": "arn:aws:apprunner:us-east-1:01234567890:observabilityconfiguration/my-observability-no-vendor/1/2bc8fefaddcc4b878c0dfe2598412555",
        "ObservabilityConfigurationName": "my-observability-no-vendor",
        "ObservabilityConfigurationRevision": 1,
        "Latest": false,
        "Status": "ACTIVE",
        "CreatedAt": "2024-06-02T09:46:36.997000+09:00"
    }
}
```

But the App Runner Service with it could not be deployed. It threw an error.
![スクリーンショット 2024-06-02 10 11 59](https://github.com/aws/aws-cdk/assets/13781813/4dae7f4d-850b-4271-851b-669cb4705d4a)



So, I added the above information to the documentation for the vendor property.